localMatchApp/logIn.cpp

```diff
@@ -0,0 +1,31 @@
+#include "logIn.h"
+
+using namespace std;
+
+logIn::logIn()
+{
+    //Connect to database.
+}
+
+logIn::~logIn()
+{
+    //disconnect to database.
+}
+
+int logIn::signUp(const string user, const string password)
+{
+  //Send information to database.
+  return 0;
+}
+
+bool logIn::isEqualPass(const string pass, const string rePass)
```

Comment:
let's use more representative names for the parameters

Reply:
Done